mappings/net/minecraft/entity/ItemEntity.mapping

```diff
@@ -3,8 +3,8 @@ CLASS net/minecraft/class_1542 net/minecraft/entity/ItemEntity
 	FIELD field_7200 thrower Ljava/util/UUID;
 	FIELD field_7201 health I
 	FIELD field_7202 pickupDelay I
-	FIELD field_7203 hoverHeight F
-	FIELD field_7204 age I
+	FIELD field_7203 uniqueOffset F
+	FIELD field_7204 itemAge I
```

Comment:
mind document this is `Age` in nbt and note the name discrepancy
```suggestion
	FIELD field_7204 itemAge I
		COMMENT The number of ticks since the item has been created. It is a short value with key {@code Age} in the NBT structure.
```